src/components/InConstruction.js

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from "react"
+import { Container } from "reactstrap"
+
+class InConstruction extends Component {
+  render() {
```

Comment:
you should call it ComingSoon and it should be a => component because it has no state